packages/create-plugin/src/utils/utils.templates.ts

```diff
@@ -96,10 +96,10 @@ export function renderTemplateFromFile(templateFile: string, data?: any) {
 export function getTemplateData(data?: Partial<TemplateData>): TemplateData {
   const { features } = getConfig();
   const currentVersion = getVersion();
-  const pluginJson = getPluginJson();
-  const pluginType = data?.pluginType ?? pluginJson.type;
-  const packageManagerInfo = getPackageManagerWithFallback();
-  const packageManagerName = data?.packageManagerName ?? packageManagerInfo.packageManagerName;
+  const pluginJson = data ? undefined : getPluginJson();
+  const pluginType = data?.pluginType ?? pluginJson?.type;
+  const packageManagerInfo = data ? getPackageManagerFromUserAgent() : getPackageManagerWithFallback();
```

Comment:
I think it was fine to leave the logic of `getPackageManagerFromUserAgent()` at the generate command - there can be cases when the `data` is provided yet it's not a generate command (e.g. in tests).

Reply:
Same is here - when it is just `const packageManagerInfo =  getPackageManagerWithFallback();` it is failing with the message "Make sure you run this command from a plugins root directory."

Reply:
Yeah, you are actually right, handling these errors differently (in case it's not run from a plugin directory) would make the code quite complicated, and it's probably also unnecessary at this point.